go/stellar/stellar.go

```diff
@@ -970,6 +970,20 @@ func FormatCurrencyWithCodeSuffix(ctx context.Context, g *libkb.GlobalContext, a
 	if err != nil {
 		return "", err
 	}
+
+	// some currencies have the same symbol as code (CHF)
+	conf, err := g.GetStellar().GetServerDefinitions(ctx)
+	if err != nil {
+		return "", err
+	}
+	currency, ok := conf.Currencies[code]
+	if !ok {
+		return "", fmt.Errorf("FormatCurrency error: cannot find curency code %q", code)
+	}
+	if currency.Symbol.Postfix && currency.Symbol.Symbol == code.String() {
+		return pre, nil
+	}
```

Comment:
hm at this point i'll probably modify the function above `func FormatCurrency` to take another parameter `addCodeSuffix bool` and do that logic. what do you think?

Reply:
I'm not a huge fan of bool parameters as when you see it in the code you have no idea what it is.

    FormatCurrency(ctx, g, amount, code, true)

no clue what true means.

Reply:
sure, it's not ideal either. if you are OK with `FormatCurrencyWithCodeSuffix` having to do this amount of boiler plate (number of LOC to get currency definition is unfortunate), then I'm fine as well